adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -54,6 +55,9 @@ def __init__(self, esp, secrets, status_pixel=None, attempts=2):
         self.attempts = attempts
         requests.set_interface(self._esp)
         self.statuspix = status_pixel
+        self._is_rgb_led = False
```

Comment:
there is no need to have an additional variable to lose/keep track of, just do all tests in the function in question